pyControl4/lock.py

```diff
@@ -0,0 +1,86 @@
+class C4Lock:
+    def __init__(self, C4Director, item_id):
+        """Creates a Control4 Lock object.
+
+        Parameters:
+            `C4Director` - A `pyControl4Ex.director.C4Director` object that
+            corresponds to the Control4 Director that the Lock is connected to.
+
+            `item_id` - The Control4 item ID of the Lock.
+        """
+        self.director = C4Director
+        self.item_id = item_id
+
+    async def is_locked(self):
```

Comment:
To match the names of functions that retrieve states in other parts of the library (like https://github.com/lawtancool/pyControl4/blob/master/pyControl4/alarm.py#L18), could these be changed to `getLockState` and `getLockStateVerified`?

Reply:
I applied these changes.